internal/pkg/flink/types/statements.go

```diff
@@ -73,10 +74,20 @@ type MockStatementResult struct {
 type StatementError struct {
 	Msg              string
 	HttpResponseCode int
+	FailureMessage   string
 }
 
 func (e *StatementError) Error() string {
-	return e.Msg
+	if e == nil {
+		return ""
+	}
+	if e.Msg == "" {
+		return "Error"
+	}
+	if e.FailureMessage == "" {
+		return fmt.Sprintf("Error: %s", e.Msg)
+	}
```

Comment:
How are these different?

Reply:
The first would just be "Error" while the second would be "Error: " if e.Msg was empty. 
Maybe "Unknown Error" or something like that would be better for the first? This case should never happen really, but just to be safe..

Reply:
`Error: No message`?

Reply:
Using that now